Can't open Microsoft Teams - blank screen

**Describe the bug**
Trying to open Microsoft Teams leaves a blank page.

**To Reproduce**
Steps to reproduce the behavior:
1. Open Microsoft Teams
2. That's it, a blank page should be shown (the error)

**Expected behavior**
Load normally

**Screenshots**
![image](https://user-images.githubusercontent.com/7959437/61073245-9769c480-a3da-11e9-8cbf-79542f19a01d.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.2
 - Installation package deb
 - Version teams-for-linux_0.4.0_amd64.deb

**Additional context**
Clicking on debug option I get this:

![image](https://user-images.githubusercontent.com/7959437/61073379-e6175e80-a3da-11e9-8c81-d7850813f6af.png)

Any idea of what's going on? Thanks.

> Added some stuff to clear the cache. Merging now in develop so it should land in snap edge channel as 0.6.2. Can you snap people test it? I haven't find any issues but I didn't had any issues before (so difficult to test)

installed the --edge version via Snap, same blank screen appears on refresh or new start. 
Console shows: 
`SettingsService: Refreshing settings. Array(1)
msteams-web-bridge.min.js:1 SettingsService: Same key already exists in lower case lookup table,             existing value: enableHidControlWhileSfBisRunning,             duplicate: key: enableHidControlWhileSfBIsRunning, value: false Array(1)
warn @ msteams-web-bridge.min.js:1
msteams-web-bridge.min.js:1 SettingsService: Refreshing settings. Array(1)
msteams-web-bridge.min.js:1 SettingsService: Same key already exists in lower case lookup table,             existing value: enableHidControlWhileSfBisRunning,             duplicate: key: enableHidControlWhileSfBIsRunning, value: false Array(1)
warn @ msteams-web-bridge.min.js:1
angular.min.js:35 Uncaught IPC object is null
angular.min.js:112 Error: Action with name notificationsService_startup already exists
    at r.registerAction (https://statics.teams.microsoft.com/hashedjs/3.1-app.min-47228058.js:2008:5774)
    at new n (https://statics.teams.microsoft.com/hashedjs/4-app.desktop.min-b52e7579.js:9:1263)
    at Object.instantiate (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:37:491)
    at Object.<anonymous> (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:38:260)
    at Object.invoke (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:37:390)
    at Object.$get (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:35:71)
    at Object.invoke (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:37:390)
    at https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:39:175
    at d (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:36:369)
    at e (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:37:96) undefined
(anonymous) @ angular.min.js:112
The Content-Security-Policy directive 'prefetch-src' is implemented behind a flag which is currently disabled.

_#/conversations/General?threadId=19:ae58acc1466447a0ad23f509f7823ac0@thread.skype&ctx=channel:1 The Content-Security-Policy directive 'prefetch-src' is implemented behind a flag which is currently disabled.

_:356 index.html: remove startup cache listeners
msteams-web-ng1-services.min.js:1 Cleaned the join location
serviceworker.js:362 Error: service-worker unable to find response in cache for : https://statics.teams.microsoft.com/hashedcss/stylesheets.theme-default.min-d45ed988.css
    at serviceworker.js:358` 

oh and now I don't know where to find "Application Cache" in order to delete it ...

--- a/app/browser/index.js
+++ b/app/browser/index.js
@@ -88,5 +88,10 @@
 		injector.get('settingsService').appConfig.hideGetAppButton = true;
 		injector.get('settingsService').appConfig.enableMobileDownloadMailDialog = false;
 	}
+	Object.defineProperty(navigator.serviceWorker, 'register', {
+		value: () => {
+			return Promise.reject()
+		}
+	});
 }());
 

--- a/app/mainAppWindow/index.js
+++ b/app/mainAppWindow/index.js
@@ -13,11 +13,10 @@ let aboutBlankRequestCount = 0;
 
 let window = null;
 
-exports.onAppReady = async function onAppReady() {
+exports.onAppReady = function onAppReady() {
 	window = createWindow();
-	await clearCache(window.webContents.session);
-	await clearStorageData(window.webContents.session);
-
+	window.webContents.session.clearCache(()=> console.log("session cache cleared"));
+	window.webContents.session.clearStorageData({ storages: ['appcache']} ,()=> console.log("appcache cleared"));
 	new Menus(window, config, iconPath);
 
 	window.on('page-title-updated', (event, title) => {
@@ -148,12 +147,4 @@ function createWindow() {
 	};
 
 	return window;
-}
-
-const clearCache = (session) => {
-	new Promise((resolve) => session.clearCache(resolve(console.log("session cache cleared"))));
-}
-
-const clearStorageData = (session) => {
-	new Promise((resolve) => session.clearStorageData({ storages: ['appcache']} , resolve(console.log("appcache cleared")) ));
 }

--- a/app/mainAppWindow/index.js
+++ b/app/mainAppWindow/index.js
@@ -15,7 +15,6 @@ let window = null;
 
 exports.onAppReady = function onAppReady() {
 	window = createWindow();
-	window.webContents.session.clearCache(()=> console.log("session cache cleared"));
 	new Menus(window, config, iconPath);
 
 	window.on('page-title-updated', (event, title) => {

--- a/app/mainAppWindow/index.js
+++ b/app/mainAppWindow/index.js
@@ -16,7 +16,6 @@ let window = null;
 exports.onAppReady = function onAppReady() {
 	window = createWindow();
 	window.webContents.session.clearCache(()=> console.log("session cache cleared"));
-	window.webContents.session.clearStorageData({ storages: ['appcache']} ,()=> console.log("appcache cleared"));
 	new Menus(window, config, iconPath);
 
 	window.on('page-title-updated', (event, title) => {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "0.6.5",
+  "version": "0.6.6",
   "main": "app/index.js",
   "description": "Unofficial client for Microsoft Teams for Linux",
   "homepage": "https://github.com/IsmaelMartinez/teams-for-linux",

